src/schema/search.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  GraphQLList,
+  GraphQLEnumType,
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLFieldConfig,
+} from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { connectionWithCursorInfo } from "schema/fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/fields/pagination"
+import { SearchableItem } from "./searchableItem"
+
+import { Searchable } from "./searchable"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+
+import { visit, BREAK } from "graphql"
+
+export const SearchIndex = new GraphQLEnumType({
+  name: "SearchIndexes",
+  values: {
+    ARTWORK: {
```

Comment:
We can add `MarketingCollection` at some point 😎 

That's going to be...interesting since thats merged in from KAWS. 👻 

The good news is (not only should we save that for a separate PR), but if there's some issues w/ that, we can skip rehydration for those types and just rely on the `... on SearchableItem` fields. Currently the ES data for collections just contains the text we're interested in searching, but it's easy to populate the image there if needed. I don't think there's any other data the collection offers that we even need (besides the title/label/image/href), so that's why I'm thinking we could just wind up skipping hydration for those types.